components/notability/wikis/ageofempires/notability_checker_config.lua

```diff
@@ -32,7 +32,7 @@ Config.MAX_NUMBER_OF_COACHES = 1
 -- Which LPDB placement parameters do we care about?
 Config.PLACEMENT_QUERY =
 	'pagename, tournament, date, placement, liquipediatier, ' ..
-	'liquipediatiertype, players, extradata, mode'
+	'liquipediatiertype, extradata, mode'
```

Comment:
Might as well move the placement query out of config, since no wiki has it adjusted (the structure doesn't even allow to access other fields)